test/WebJobs.Script.Tests/FileMonitoringServiceTests.cs

```diff
@@ -75,7 +123,8 @@ public static async Task TestAppOfflineDebounceTime(string fileName, int delayIn
                 {
                     RootLogPath = tempDir,
                     RootScriptPath = tempDir,
-                    FileWatchingEnabled = true
+                    FileWatchingEnabled = true,
+                    WatchFiles = { "host.json" }
```

Comment:
This isn't needed, right? Please remove.

Reply:
This is needed because in the unit test, it expects a restart event to be published when "host.json" is changed. Which is the default behavior as the options will always have "Watchfiles" with "host.json".

Reply:
Ah yes. My point was that we want to ensure that the default file watch list doesn't change, but that's handled by options setup. This unit test bypasses options setup so must specify explicitly.